### PR TITLE
Add Jenkins file

### PR DESCRIPTION
**Pull request by fpapon**
Add 2 Jenkins file for the new Apache CI:

- https://ci-builds.apache.org/job/Unomi/job/Unomi-jdk8/
- https://ci-builds.apache.org/job/Unomi/job/Unomi-jdk11/

A new job will be automatically created for every PR. 
However, the PR has to rebase to have the Jenkin files to make it work.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                    junit(testResults: '**/surefire-reports/*.xml', allowEmptyResults: true)
+                    junit(testResults: '**/failsafe-reports/*.xml', allowEmptyResults: true)
+                }
+            }
+        }
+
+//        stage('Integration tests') {
+//            steps {
+//                echo 'Running integration tests'
+//                sh 'mvn test -Pci-build-itests'
```
Comment (sergehuber) [inline Jenkinsfile:95]: We can activate this if we change it to : 
```suggestion
//                sh 'mvn clean install -P integration-tests'
```

File: Jenkinsfile-jdk11 [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                    junit(testResults: '**/surefire-reports/*.xml', allowEmptyResults: true)
+                    junit(testResults: '**/failsafe-reports/*.xml', allowEmptyResults: true)
+                }
+            }
+        }
+
+//        stage('Integration tests') {
+//            steps {
+//                echo 'Running integration tests'
+//                sh 'mvn test -Pci-build-itests'
```
Comment (sergehuber) [inline Jenkinsfile-jdk11:95]: Same here:
```suggestion
//                sh 'mvn clean install -P integration-tests'
```

Comment (sergehuber) [reply Jenkinsfile-jdk11:95]: Thanks for the PR, looks good I just added some suggestions to be able to activate the integration tests which I think would be a good thing.

Comment (fpapon) [reply Jenkinsfile-jdk11]: @sergehuber I updated with this `mvn -pl itests clean install -P integration-tests` to not clean install all the project.

**Review by Taybou — APPROVED** [review]

**Review by sergehuber — APPROVED** [review]

**Pull request merged**